vision-quickstart/app/src/main/java/com/google/mlkit/vision/demo/VisionProcessorBase.java

```diff
@@ -193,13 +182,12 @@ public void processImageProxy(ImageProxy image, GraphicOverlay graphicOverlay) {
         }
         if(bufferImageList.size()>depth)
             bufferImageList.clear();        // O(n) operation
-
-        bufferImageList.add(new Pair(toBitmap(image.getImage()), System.currentTimeMillis()));
+        bufferImageList.add(new bufferImage(BitmapUtils.getBitmap(image), image.getImageInfo()));
```

Comment:
Did you forget to add the file for BitmapUtils?

Also Utils is usually a bad name for things, try to name it for what it actually does, otherwise you end up with giant classes containing lots of unrelated things.  In this case BitmapConverter or something is probably more appropriate.

Reply:
`BitmapUtils` was already present in the boilerplate. I just didn't notice the existence of such function. Found about it after looking into it's code.